packages/cardhost/app/templates/components/cardhost-modal-target.hbs

```diff
@@ -0,0 +1,12 @@
+{{#from-elsewhere name="cardhost-modal" as |args outsideParams|}}
+  {{#if args}}
+    <div class="cardhost-modal-backdrop"></div>
+    <div
+      role="button"
```

Comment:
We should not use `<div role=button>`. We lose all the automatic keyboard navigation and focus management that the browser gives you. Can we make it a `<button>`?

Reply:
sure...?